src/js/slider.js

```diff
@@ -0,0 +1,150 @@
+import { changeSelectedMarkers } from "./filter";
```

Comment:
maybe call the file `populationSlider.js` so that it's more self-documenting what this thing is about. Ditto the other files.